requirements.txt

```diff
@@ -7,5 +7,5 @@ django-debug-toolbar==3.4.0
 djangorestframework==3.13.1
```

Comment:
Do not add the content of the `media/` folder and the `media/` folder itself to the PR. Remove it from the PR.